### PR TITLE
Change from arr.mean to np.mean(arr) fixes #1164

### LINKED ISSUE #1164: 'tensorflow.python.framework.ops.EagerTensor' object has no attribute 'mean'
Hi, guys hope everyone is safe and healthy! I'm trying to get SHAP values from a Keras model where after loading the model and weights, I import the CSV data using pandas then got a slice as follows;

`background = df_full.iloc[np.array(df_full.index.array)[np.random.choice(df_full.shape[0], 10, replace=False)]]`
`background = background.reset_index(drop=True)`
`explainer = shap.DeepExplainer(model,tf.constant(background.values, dtype=np.float32))`
`X  = df_full.iloc[np.array(df_full.index.array)[np.random.choice(df_full.shape[0], 
                                                                 10, replace=False)]]`
`shap_values = explainer.shap_values(tf.constant(X.values, dtype=np.float32))`

But I get an error from `shap/explainers/deep/deep_tf.py` L308 saying as follows. 

`'tensorflow.python.framework.ops.EagerTensor' object has no attribute 'mean'`

In `deep_tf.py`, I found the corresponding line which was;

`phis[l][j] = (sample_phis[l][bg_data[l].shape[0]:] * (X[l][j] - bg_data[l])).mean(0)`

where this `.mean(0)` was problematic so I changed it with the following;

`phis[l][j] = np.mean(sample_phis[l][bg_data[l].shape[0]:] * (X[l][j] - bg_data[l]), axis=0)`

Now it's working for me but I wanted to ask if I'm doing any mistake here, looks like fine and results are as expected. Would be appreciated if you can shine some light on the matter. 

Cheers

### PR DESCRIPTION
**Pull request by Mahdisadjadi**
Change the averaging method from `array.mean(0)` to `numpy.mean(array, axis=0)` to fix #1164 .

### CODE REVIEW

**Comment by slundberg** [comment]
Hey! I am sorry this got missed a while back. I am happy to merge this, but the file as moved to `_deep/deep_tf.py`, do you mind updating this PR to match that? Otherwise if I make the change it won't give you authorship credit when I merge :)

**Comment by lrjball** [comment]
LGTM

**Comment by bmwise14** [comment]
Can this fix be merged in the next version of shap? I am using shap==0.37.0 with tensorflow==2.3.1. It does not appear that the error in deep_tf.py has been resolved yet.

**Comment by skiiz** [comment]
> Hey! I am sorry this got missed a while back. I am happy to merge this, but the file as moved to `_deep/deep_tf.py`, do you mind updating this PR to match that? Otherwise if I make the change it won't give you authorship credit when I merge :)

Hi, was this fixed in deep_tf.py 
there is still
                ```   phis[l][j] = (sample_phis[l][bg_data[l].shape[0]:] * (X[l][j] - bg_data[l])).mean(0) ```
